kubernetes/client/ws_client.py

```diff
@@ -23,22 +25,17 @@
 
 class WSClient:
     def __init__(self, configuration, url, headers):
```

Comment:
I will add some documentation for methods in this class. __init__ specifically does not need docs in my opinion as this class is only created inside this file.